backend/src/main/java/com/onlinebookstore/model/Book.java

```diff
@@ -26,4 +30,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(nullable = false)
```

Comment:
this is default value, why others are without annotation and this one is with it?